src/components/Input/Button.tsx

```diff
@@ -14,6 +15,7 @@ type Props = {
   loading?: boolean;
   loadingSize?: number;
   onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  place?: PlacesType;
```

Comment:
Rename to `tooltip-place`

Reply:
Used `tooltipPlace` to avoid a hyphen in the property names, should be close enough I think